tests/files/params_config/dataclass_deps.yaml

```diff
@@ -0,0 +1,13 @@
+MD:
+  steps: 100
+  thermostat:
+    temperature: 300
+multiple_deps_MD:
+  steps: 100
+  thermostat:
+    part_0:
+      temperature: 300
+    part_2:
+      temperature: 400
```

Comment:
other parameters are also not affected by the other, e.g. `params=[1,2,3]` is the same as `params=[2,1,3]` so probably we can make this a simple list?

Reply:
In case of ASEMD, are we even planning to support lists of thermostats? I would argue that the order of entries in a params=list most often do affect the outcome, since outputs of e.g. the DFT calculator nodes will have the same order.

Reply:
ahh I see the problem. Yes I would say this is a dvc issue